src/UnitfulRecipes.jl

```diff
@@ -86,24 +86,45 @@ function _resolve_axis!(attr, arr, T, axis)
     return arr
 end
 
-struct Literally
+"""
+    ProtectedString
+
+Wrapper around a `String` to "protect" it from `recipe!` passes.
+
+TODO: Decide the name. I suggest `ProtectedString`.
+Note that I think the name is not too important at this stage,
+because we can use a string macro (see `P_str` below)
+"""
+struct ProtectedString <: AbstractString
     content::String
 end
+# Minimum required AbstractString interface to work with Plots?
+const S = ProtectedString
+Base.iterate(n::S) = iterate(n.content)
+Base.iterate(n::S, i::Integer) = iterate(n.content, i)
+Base.codeunit(n::S) = codeunit(n.content)
+Base.ncodeunits(n::S) = ncodeunits(n.content)
+Base.isvalid(n::S, i::Integer) = isvalid(n.content, i)
+Base.pointer(n::S) = pointer(n.content)
+Base.pointer(n::S, i::Integer) = pointer(n.content, i)
+# macro for easy-to-use interface?
+# i.e., so that `U"foo"` creates `ProtectedString("foo")`
```

Comment:
I meant P"foo" here, corrected locally and will push after our next chat if that's OK 😃